.github/workflows/ci-v3.yml

```
@@ -23,7 +23,7 @@ jobs:
           node-version: ${{ matrix.node }}
       - name: Install dependencies
         uses: bahmutov/npm-install@v1
-      - run: yarn test:ci
+      - run: yarn test:ci --base=${{ github.event.before }}
```

I don't think we need to change something here. This will only run on the v3 branch which doesn't know nx

```suggestion
      - run: yarn test:ci
```